packages/console-types/src/types/catalog/well-known-items/application/collection.ts

```diff
@@ -84,26 +87,138 @@ const fieldsSchema = {
   type: 'array',
 } as const satisfies JSONSchema
 
+const internalEndpointsSchema = {
```

Comment:
Those schemas are different from the one used in the config collections?

Reply:
A little

This one has a limitation on max and min number of items (which I suppose we can replicate on the other), and it does not have the `defaultSortingIndex` property (that is not considered by Console when creating a collection for marketplace)

I think we can merge the two interfaces pretty safely, though